python/hail/api1/context.py

```diff
@@ -132,6 +132,9 @@ def _set_history(self, history):
         assert not self._history_was_set, "Cannot set history for HailContext more than once."
         self._history = history.set_varid("hc")
         self._history_was_set = True
+
+    def _get_temp_file(self, n_char=10, prefix=None, suffix=None):
```

Comment:
I don't think this should go here. It should go in utils/misc.py. You can get the current hail context with `Env.hc()._jhc`.

Reply:
Ahh, great!